src/styles/services.scss

```diff
@@ -1,24 +1,36 @@
 .services {
+  @media (min-width: 700px) {
     padding-top: 5rem;
+  }
+
+  h4 {
+    margin: 1rem 0;
+    text-align: center;
+
+    @media (min-width: 700px) {
+      font-size: 3rem;
+      padding-top: 8rem;
+    }
+  }
 }
 
 .service-container {
-    padding: 10rem 1rem 15rem 1rem;
-    display: flex;
-    justify-content: space-evenly;
-    text-align: center;
+  display: flex;
+  text-align: center;
 }
 
 .service-card {
-    width: 20%;
-    img {
-        width: 4.5rem;
-        height: 4.5rem;
-        display: grid;
-        margin: auto;
-    }
-    h3 {
-        margin-left: 1rem;
-        padding: 1rem;
-    }
+  @media (min-width: 700px) {
+    flex: 1;
+    padding: 10rem 4rem;
+  }
```

Comment:
☝️ Removing the initial `width: 20%` means these will be full width on a mobile device. Using `flex: 1` inside the media query is like using `1fr` in grid. It says make all these containers the same size and fill the available space. 

Padding limits the width of the content inside the container to get the effect you were looking for with the 20% width.

Images will be sizes to fit the container.